src/components/VoteTracker.jsx

```diff
@@ -0,0 +1,38 @@
+import {FaArrowUp, FaArrowDown} from 'react-icons/fa'
+import { useEffect, useState } from "react";
+import { patchVotes } from "../utils/api";
+import styles from "../styles/VoteTracker.module.css";
+export const VoteTracker = ({ votes, articleId }) => {
+  const [currentVotes, setCurrentVotes] = useState(votes);
+
+  const handleUpvoteClick = () => {
+    setCurrentVotes((oldVotes) => {
+      return oldVotes + 1;
+    });
+    patchVotes(articleId, 1).then((article) => {
+      console.log(article);
+    }).catch((err) => {
+        setCurrentVotes((oldVotes) => {
+            return oldVotes - 1;
+          });
+    });
+  };
+  const handleDownvoteClick = () => {
+    setCurrentVotes((oldVotes) => {
+      return oldVotes + -1;
+    });
+    patchVotes(articleId, -1).then((article) => {
+      console.log(article);
+    }).catch((err) => {
+        setCurrentVotes((oldVotes) => {
+            return oldVotes + 1;
+          });
+    });
+  };
+  return (
+    <div className={styles.VoteTracker}>
+      Votes: {currentVotes} <button onClick={handleUpvoteClick}><FaArrowUp/></button>
+      <button onClick={handleDownvoteClick}><FaArrowDown/></button>
+    </div>
+  );
+};
```

Comment:
This looks good, Nathan, nice use of optimistic rendering. I'd advise adding some kind of feedback to the user here if the voting is unsuccessful. Don't forget to remove any unnecessary console.logs.